package.json

```diff
@@ -163,7 +163,7 @@
     ]
   },
   "dependencies": {
-    "@gnosis.pm/safe-apps-sdk": "0.4.2",
+    "@gnosis.pm/safe-apps-sdk": "https://github.com/gnosis/safe-apps-sdk#a6c168b",
```

Comment:
should we release a version with the new send transaction type to npm first?

Reply:
That's a good question. I haven't thought much about it, but I didn't release it because of the possible changes after the review. Maybe we can release it like a `beta` or `rc`, but I wouldn't release a new version for each ticket